### DISCUSSION TITLE
After updating to 1.3.5, when connecting via LAN IP, it disconnects and reconnects every 8 seconds or so.

### DISCUSSION THREAD

**Discussion by hwf1324** [rustdesk/rustdesk#10221] [General]
### Bug Description

After I updated RustDesk to 1.3.5 on both devices today.

After I establish a connection via LAN IP, it disconnects and re-establishes the connection about every 8 seconds or so. And it will pop up the prompt in the screenshot.

### How to Reproduce

Connecting via LAN IP, I haven't tested this on other machines and network environments.

### Expected Behavior

No connection will be broken.

### Operating system(s) on local (controlling) side and remote (controlled) side

Windows 11 -> Windows 10

### RustDesk Version(s) on local (controlling) side and remote (controlled) side

1.3.5 -> 1.3.5

### Screenshots

![图片](https://github.com/user-attachments/assets/19b477f1-1a59-4d88-b5be-dc580249027e)

**Comment by rustdesk** [comment]
@21pages

**Comment by 21pages** [comment]
Could you confirm whether the controlled end crashed （`rustdesk.com --server` process) every 8 seconds? You can confirm by 
1. Task manager
2. Controlled side log, each process creates one log file, https://github.com/rustdesk/rustdesk/wiki/FAQ#access-logs

**Comment by qicheng02** [comment]
before install   you should  delete all  old version documents .  i have been  got the same thins

**Reply by 21pages** [reply]
> delete all old version documents

What does this mean? Remove config files or remove old version program.

**Comment by qicheng02** [comment]
fist ,uninstall the old program   ,then,install  the latest version of rustdesk ,  try it

**Reply by qicheng02** [reply]
if done all  , the falut is still exist  then  go back  1.33 (maybe the best version)

**Comment by hwf1324** [comment]
The issue has been resolved.

In fact, it might be best to shut down the RustDesk service before upgrading RustDesk.

Maybe there are some issues with upgrading when the service is not turned off.

For example, when I upgraded RustDesk on my host computer, after opening the downloaded `.exe` program without shutting down the service, there was no install button for the upgrade.
But after closing the service it shows the install button for the upgrade.

On the remote machine, although I don't encounter the above problem, I encounter this problem after upgrading.
